x-pack/plugins/reporting/server/export_types/printable_pdf_v2/lib/get_custom_logo.test.ts

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
`get_custom_logo` should also be under `export_types/common` so v1 and v2 of PDF export can share it.